<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1941</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pizza House</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/titleicon.jpeg</normaloff>../img/titleicon.jpeg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>39</y>
      <width>1840</width>
      <height>951</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
background-color: rgb(55, 55, 55);
border-radius: 50px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>60</y>
       <width>1011</width>
       <height>831</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(91, 88, 88);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Cart</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>70</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Contact No.</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cname">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
border-radius: 15px;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ccontact">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>120</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
border-radius: 15px;
}</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_dinein">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Dine-in</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_take">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>170</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Takeaway</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_del">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>170</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Delivery</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>210</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cadd">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>260</y>
        <width>881</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
border-radius: 15px;
}</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_cp">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>310</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Cash Payment</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_op">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>310</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Online Payment</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>360</y>
        <width>871</width>
        <height>371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(174, 173, 173);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_order">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>770</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(255, 159, 51);
	color: rgb(0, 0, 0);
border-radius: 20px;
}</string>
      </property>
      <property name="text">
       <string>Place Order</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_pizza">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(255, 159, 51);
	color: rgb(0, 0, 0);
border-radius: 20px;
}</string>
     </property>
     <property name="text">
      <string>Pizza</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_pizzam">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>150</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(255, 159, 51);
	color: rgb(0, 0, 0);
border-radius: 20px;
}</string>
     </property>
     <property name="text">
      <string>Pizza Mania</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_dsides">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>700</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(255, 159, 51);
	color: rgb(0, 0, 0);
border-radius: 20px;
}</string>
     </property>
     <property name="text">
      <string>Delicious Sides</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_dessert">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>700</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(255, 159, 51);
	color: rgb(0, 0, 0);
border-radius: 20px;
}</string>
     </property>
     <property name="text">
      <string>Dessert</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_bev">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>410</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(255, 159, 51);
	color: rgb(0, 0, 0);
border-radius: 20px;
}</string>
     </property>
     <property name="text">
      <string>Beverages</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_showempid">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>EMP ID</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
